src/vm/eventpipeprovider.h

```diff
@@ -97,14 +99,15 @@ class EventPipeProvider
     // Unset the provider configuration for the specified session (disable sets of events).
     // This is called by EventPipeConfiguration.
     EventPipeProviderCallbackData UnsetConfiguration(
+        INT64 keywordsForAllSessions,
+        EventPipeEventLevel providerLevelForAllSessions,
         uint64_t sessionId,
         INT64 keywords,
         EventPipeEventLevel providerLevel,
         LPCWSTR pFilterData);
 
     // Refresh the runtime state of all events.
     void RefreshAllEvents(
-        uint64_t sessionId,
```

Comment:
Synced with you offline - We should remove the other two arguments since they're not used either.

Reply:
Agreed, this is a left over ... I would have expected a warning treated as error for unused variables/params :(

Reply:
Will be addressed in the next iteration.